src/UI.py

```diff
@@ -50,25 +51,15 @@ def checksources(self):
                 IDs.append(self.newsSources[0][i])
         return IDs
 
-    def generateHeadlines(self): 
+    def generateTracks(self): 
         headlines = makeRequest(self.checksources())
-        self.headlines['text'] = ""
+        strHeadline = ''
         for item in headlines:
-            self.headlines['text'] += item + "\n"
-
-def checkKeywords():
-    # txt = self.exampleTextInput.get()
-    # print(txt)
-    with open('data_file2.json'):
-        for line in file.readlines:
-            getTitles()
-        if x == re.search(txt):
-            print("yes")
-        else:
-            print("no")
-
-
-
```

Comment:
possible indentation problem with strHeadline += item + ' 'being in the else function